inc/functions/functions-templates.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * Display Authors
+ */
+function displayAuthors()
```

Comment:
Name convention should follow WP coding standards
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions